### DISCUSSION TITLE
Most effective way to create new ContentTypeSettings that display in the same area?

### DISCUSSION THREAD

**Discussion by willnationsdev** [OrchardCMS/OrchardCore#8560] [Q&A] [answered]
Similar to "Creatable", "Listable", "Securable", etc. I want to create my own custom settings like, e.g. "Downloadable" which generates an Excel spreadsheet download of the ContentItems.

To my understanding, I could write an extension method for the ContentTypeDefinitionBuilder that would add my own custom settings to the ContentTypeDefinition and then implement a custom ContentTypeDefinitionDisplayDriver that would allow me to specify a user-defined shape type to append to the rendered content of the page.

The problem, at that point, becomes how to ensure that my shape template HTML shows up below the other "ContentTypeSettings_Edit" fields. I assume I would need to take advantage of the placement API, but since everything is at `Content:5`, I feel like I would need to manually adjust the placement of all of the other shapes on the page to something like `Content:7`, just so that I can put my new ContentTypeSettings at `Content:6`.

Is there a simpler way to just append my shape to another shape?

**Comment by jtkech** [comment, marked as answer]
Try e.g. `Content:5.5` ;)

**Reply by willnationsdev** [reply]
Ha! Well, that was definitely easier than expected. Gotta love OrchardCore sometimes. Geez. Thanks!

**Reply by willnationsdev** [reply]
@jtkech With that said, it might be a good idea for someone to update the Placement docs to communicate that float values are supported there. I had the impression it only used integers.

**Reply by jtkech** [reply]
In fact it only compare integers, but all those that are separated by a dot `.` one after each other, until one is greater or lower
